node/src/components/consensus/protocols/highway.rs

```diff
@@ -462,6 +461,33 @@ impl<I: NodeIdT, C: Context + 'static> HighwayProtocol<I, C> {
             .map_or(false, is_switch)
     }
 
+    // Check if we've made any progress since joining.
+    // If we haven't, we might have been left alone in the era and we should request the state from
+    // peers.
+    fn handle_progress_alert_timer(&mut self, now: Timestamp) -> ProtocolOutcomes<I, C> {
+        if self.evidence_only || self.finalized_switch_block() {
+            return vec![]; // Era has ended. No further progress is expected.
+        }
+        if self.last_panorama == *self.highway.state().panorama() {
+            // We haven't made any progress. Request latest panorama from peers and schedule
+            // standstill alert. If we still won't progress by the time
+            // `TIMER_ID_STANDSTILL_ALERT` is handled, it means we're stuck.
+            let mut outcomes = self.latest_panorama_request();
+            outcomes.push(ProtocolOutcome::ScheduleTimer(
+                now + self.standstill_timeout * 2,
```

Comment:
I'm not sure about this `*2` here. I mean, we did discuss making the wait after the initial check longer, but this is a recurring timer that will be firing every `self.standstill_timeout`, anyway, so I'm not sure if it makes much sense to make only this single period longer.
Not a blocker, though.

Reply:
Yeah, I wasn't convinced either (didn't have a strong opinion about it) but wanted to take your suggestion into account. Will revert it.

Reply:
Removed in [`f3b1a5c` (#1397)](https://github.com/casper-network/casper-node/pull/1397/commits/f3b1a5c25b031cd6c04328b2e84d863a18c5a2d1)